docs/commands/json.merge.md

```diff
@@ -1 +1,120 @@
-Sets or updates the JSON value at a path.
+Merge one or more values in a JSON document at `path` in `key`
```

Comment:
Suggested:

Merges a given JSON value into matching paths. Consequently, JSON values at matching paths are updated, deleted, or expended with new children

Reply:
Only change `expend` to `expand`